### PR TITLE
tophy

### PR DESCRIPTION
**Pull request by tophyrakotomavo**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,55 +1,21 @@
 import { type NextPage } from "next";
```
Comment (TchiRubick) [inline src/pages/index.tsx:2]: Import `React` missing

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,55 +1,21 @@
...
-import Head from "next/head";
-import Link from "next/link";
-
-import { api } from "../utils/api";
 
 const Home: NextPage = () => {
-  const hello = api.example.hello.useQuery({ text: "from tRPC" });
 
   return (
     <>
```
Comment (TchiRubick) [inline src/pages/index.tsx]: Replace by a `div`

**Review by TchiRubick — CHANGES_REQUESTED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (tophyrakotomavo) [reply src/pages/index.tsx:2]: finish

File: src/pages/index.tsx (hunk 2 above)
Comment (tophyrakotomavo) [reply src/pages/index.tsx]: fnish

**Review by TchiRubick — APPROVED** [review]

**Pull request merged**